shell/install-dart-sdk.sh

```diff
@@ -17,9 +14,15 @@ wget http://storage.googleapis.com/dart-archive/channels/stable/release/latest/s
 
 echo "Downloading done."
 
+# remove the currently install version of the dart sdk
+rm -r /usr/local/bin/dart-sdk
```

Comment:
here we can remove the dart-sdk directory so we can unzip the potentially new version to the same directory name.